docs/en/ingest-management/elastic-agent/configuration/providers/kubernetes-provider.asciidoc

```diff
@@ -189,6 +189,9 @@ The available keys are:
 
 
 These are the fields available within config templating. The `kubernetes.*` fields will be available on each emitted event.
+`kubernetes.labels.*` and `kubernetes.annotations.*` used in config templating are not dedoted and should not be confused with
```

Comment:
this sentence is in the same paragraph with the previous one - https://observability-docs_2552.docs-preview.app.elstc.co/guide/en/fleet/master/kubernetes-provider.html

can we split it? and maybe highlight with `NOTE:` similar to https://github.com/elastic/observability-docs/pull/2552/files#diff-62a97949a501942a1b602f35ca893cebef309cffa0e97dd233dc6c7492a5d8f3R93?

Reply:
Yes you are right. I will highlight with NOTE

Reply:
https://github.com/elastic/observability-docs/pull/2552/commits/e7771ca322ef8a0a7b1c8cf1cc2493aa62c951d4